src/react-apps/applications/shared/src/navigation/drawer/tabletDrawerMenustyle.ts

```diff
@@ -0,0 +1,51 @@
+import { createStyles, Theme } from '@material-ui/core/styles';
+
+const drawerWidth = '240';
+
+export const styles = (theme: Theme) => createStyles({
```

Comment:
does this need the `theme`-parameter?

Reply:
Ya, because we are using it for transition for drawer